content/attacks/posts/2023-07-10-Arcadia-Finance.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2023-07-10
+target-entities: Arcadia Finance
+entity-types:
+  - DeFi
+  - Lending Platform
+attack-types: Smart Contract Exploit
+title: "Arcadia Finance Suffers $455,000 Security Breach"
+loss: 455000
+---
+
+## Summary
+
+On July 10, 2023, Arcadia Finance, a DeFi protocol on Ethereum and Optimism, experienced a significant security breach due to vulnerabilities in its smart contract. The incident resulted in a financial loss of approximately $455,000. The breach was due to inadequate security measures in the protocol's contract, allowing an attacker to manipulate the system for unauthorized asset transfers.
+
+## Attackers
+
+The identity of the hackers who attacked Arcadia Finance is unknown.
+
+Hacker Wallets:
+
+- [0xd3641c912a6a4c30338787e3c464420b561a9467](https://optimistic.etherscan.io/address/0xd3641c912a6a4c30338787e3c464420b561a9467)
+- [0x5c75e94dd0ab9c10bfd1b8073dafef031d3c050d](https://etherscan.io/address/0x5c75e94dd0ab9c10bfd1b8073dafef031d3c050d)
+
+## Losses
+
+The total loss from the Acradia Finance hack amounted to approximately $455,00, distributed across the following networks:
+
+- [Ethereum](https://phalcon.blocksec.com/explorer/tx/eth/0xefc4ac015069fdf9946997be0459db44c0491221159220be782454c32ec2d651):
```

Comment:
Don't need to duplicate links already provided in the article. In the timeline section there are both transactions.